### PR TITLE
Write default log to stdout

### LINKED ISSUE #362: Log to stdout during continuous integration
To make CI messages more informative, we should log to stdout.

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/sql-machine-learning/sqlflow/issues/362

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -6,4 +6,4 @@ service mysql start
 go generate ./...
 go get -v -t ./...
 go install ./...
-go test -v ./...
+SQLFLOW_log_dir= go test -v ./...
```
Comment (weiguoz) [inline scripts/test.sh]: I'm not sure what the effect is.
Maybe we just run `go test -v ./...`, because the default `SQLFLOW_log_dir` is ""

Comment (tonyyang-svail) [reply scripts/test.sh]: Shall we also add `SQLFLOW_log_level=debug`?

Comment (typhoonzero) [reply scripts/test.sh]: Thanks @weiguoz @tonyyang-svail , Done.

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**